tools/mavlogdump.py

```diff
@@ -76,6 +76,10 @@
     yappi.start()
 
 if args.format == 'mat':
+    # Check that the mat_file argument has been specified
+    if not args.mat_file:
+        print("mat_file argument must be specified when mat format is selected")
+        quit()
```

Comment:
I'm not familiar with this, but a quick bit of research shows use of this shouldn't be used in scripts, only in REPL.

Reply:
Changed to use "if args.mat_file is None"

Reply:
That's not the right resolution for my query :-)

Reply:
> Ah! In that case I did not understand the query at all....! Are you talking about the use of 'quit()' command, instead of something like exit() or exit(1)? If so, I used it because similar error checking in the same script was already using it. If that's not what you were talking about, then I'm lost!

Yes, quit() shouldn't be used in scripts, FWIU.

But hey, this is no worse than the script already is, so merging.

Reply:
OK thanks! Cleaning all the quits up would be a nice tidy-up, so I can do that too when I get another mo.